src/pages/add/index.vue

```diff
@@ -1,31 +1,114 @@
 <template>
-  <article class="preview">
-    <section class="preview__content">
-      <Button @click="clickHandler" size="small">Back to Home</Button>
-    </section>
-  </article>
+  <div class="add">
+    <PageHeader>
+      <template #left-btn>
+        <div class="header__left-btn">
+          <IconButton
+            @click="toggleSidebar"
+            size="large"
+            color="normal"
+            iconName="menu"
+          ></IconButton>
+        </div>
+      </template>
+      <template #title>授業の追加</template>
+    </PageHeader>
+    <div class="main">
+      <div class="main__card">
+        <CardAdd
+          @click-next-button="$router.push('/')"
+          iconName="search"
+          heading="授業の検索"
+          text="ワードや条件を指定して授業を検索・追加します。"
+        ></CardAdd>
+      </div>
+      <div class="main__card">
+        <CardAdd
+          @click-next-button="$router.push('/')"
+          iconName="get_app"
+          heading="Twinsからインポート"
+          text="Twinsで登録した授業をモジュール単位でインポートします。"
+        ></CardAdd>
+      </div>
+      <div class="main__card">
+        <CardAdd
+          @click-next-button="$router.push('/')"
+          iconName="insert_drive_file"
+          heading="CSVファイルから追加"
+          text="CSVファイルに入っている授業データを追加します。"
+        ></CardAdd>
+      </div>
+      <div class="main__card">
+        <CardAdd
+          @click-next-button="$router.push('/')"
+          iconName="style"
+          heading="手動で授業を作成"
+          text="手入力でカスタマイズして授業を作成します。"
+        ></CardAdd>
+      </div>
+    </div>
+  </div>
 </template>
 
 <script lang="ts">
+import CardAdd from "~/components/CardAdd.vue";
+import IconButton from "~/components/IconButton.vue";
+import PageHeader from "~/components/PageHeader.vue";
 import { defineComponent } from "vue";
-import { useRouter } from "vue-router";
-import Button from "../../components/Button.vue";
+import { useStore } from "~/store";
 
 export default defineComponent({
   name: "Add",
   components: {
-    Button,
+    CardAdd,
+    IconButton,
+    PageHeader,
   },
   setup: () => {
-    const router = useRouter();
+    const store = useStore();
 
-    const clickHandler = async () => {
-      await router.push("/");
+    /** ヘッダー */
+    const toggleSidebar = () => {
+      store.commit("setSidebar", !store.state.sidebar);
     };
 
-    return { clickHandler };
+    return {
+      CardAdd,
+      IconButton,
+      PageHeader,
+      toggleSidebar,
+    };
   },
 });
 </script>
 
-<style lang="scss" scoped></style>
+<style lang="scss" scoped>
+@import "~/scss/main.scss";
+
+.add {
+  margin: $spacing-0 $spacing-0;
+  @include tab-and-pc {
+    margin: $spacing-0 $spacing-9;
+  }
+  max-width: $max-pc-item-width;
+}
+
+.header__left-btn {
+  @include wide-screen {
```

Comment:
```suggestion
  @include landscape {
```
wide-screenはmin-heightも条件に含まれますが、この部分で必要な判定は”横向きかどうか"のみなので